.github/workflows/pullrequest.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and run unit tests 
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '3.1.100'
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
```

Comment:
>-configuration Release [](start = 25, length = 22)

Is there any value in building Debug? If we use things like `#if DEBUG` in the code, the two can differ so building both ensures both modes work.

Also, would we want to build both x64 and x86? Or do we just build x64 now?

Reply:
I think we start with just x64 and release, it's very rare for us to deploy debug on server side code these days.

Reply:
I've updated the pipeline to use both Release and Debug. About platform, I don't think it's needed for now since it's a library repo and all projects should be built for AnyCPU or locked to their platform withing AnyCPU solution profile.